<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <layout class="QVBoxLayout" name="verticalLayout">
                    <item>
                        <layout class="QHBoxLayout" name="Top">
                            <item>
                                <layout class="QVBoxLayout" name="LeftPanel">
                                    <item>
                                        <widget class="QLabel" name="label">
                                            <property name="text">
                                                <string>Input file</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="txtInput"/>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnBrowse">
                                            <property name="text">
                                                <string>Browse</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="Line" name="line_2">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="text">
                                                <string>Range selection</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="txtRange"/>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_3">
                                            <property name="text">
                                                <string>Output files prefix</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLineEdit" name="txtOutput"/>
                                    </item>
                                    <item>
                                        <widget class="Line" name="line_3">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="btnSplit">
                                            <property name="text">
                                                <string>Split PDF</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="verticalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Vertical</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>20</width>
                                                    <height>40</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <spacer name="horizontalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <widget class="Line" name="line">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <layout class="QVBoxLayout" name="RightPanel">
                                    <item>
                                        <widget class="QLabel" name="label_4">
                                            <property name="text">
                                                <string>Output preview</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPlainTextEdit" name="txtPreview">
                                            <property name="readOnly">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </item>
                    <item>
                        <layout class="QHBoxLayout" name="Bottom">
                            <item>
                                <widget class="QProgressBar" name="progressBar">
                                    <property name="value">
                                        <number>0</number>
                                    </property>
                                    <property name="textVisible">
                                        <bool>true</bool>
                                    </property>
                                </widget>
                            </item>
                            <item alignment="Qt::AlignRight">
                                <widget class="QPushButton" name="btnBack">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Back</string>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
